Library/Homebrew/utils/path.rb

```diff
@@ -0,0 +1,16 @@
+# typed: true
+# frozen_string_literal: true
+
+module Utils
+  module Path
+    module_function
```

Comment:
Similar to above: I recommend removing this, making the methods class defs (`def self.`), removing the `.rbi` file, and making this `typed: strict`. (I'd like to have aspects of this enforced by rubocop, but there are a number of existing violations to clean up.)